x-pack/plugins/observability/server/assets/common.ts

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
@kdelemme what do you think about naming the file as `constants` instead?

Reply:
Works for me 👍🏻